src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/PollingEventEmitterHandler.java

```diff
@@ -28,7 +30,12 @@ public void handleTask(ExternalTask externalTask) {
         log.info("Job '{}' found {} case(s)", externalTask.getTopicName(), cases.size());
         cases.stream()
             .map(caseDetailsConverter::toCaseData)
-            .forEach(mappedCase -> eventEmitterService.emitBusinessProcessCamundaEvent(mappedCase, true));
+            .forEach(mappedCase -> {
+                log.info(format("Emitting %s camunda event for case through poller: %d",
+                                mappedCase.getBusinessProcess().getCamundaEvent(),
+                                mappedCase.getCcdCaseReference()));
```

Comment:
Consider the following, for consistency with other areas you touched:

```
log.info("Emitting {} camunda event for case through poller: {}", mappedCase.getBusinessProcess().getCamundaEvent(), mappedCase.getCcdCaseReference());
```

Also, I'm seeing in other areas there's a null check for `getBusinessProcess()`. Could it be needed here as well?